src/SourceBuild/content/repo-projects/wpf.proj

```diff
@@ -8,6 +8,7 @@
     <BuildArgs>$(BuildArgs) $(FlagParameterPrefix)warnAsError $(ArcadeFalseBoolBuildArg)</BuildArgs>
     <!-- TODO setting Platform shouldn't be necesary: https://github.com/dotnet/source-build/issues/4314 -->
     <BuildArgs>$(BuildArgs) /p:Platform=$(TargetArchitecture)</BuildArgs>
+    <BuildArgs>$(BuildArgs) /p:UsingToolMicrosoftNetCompilers=false</BuildArgs>
```

Comment:
Also try `BuildWithNetFrameworkHostedCompiler`

Reply:
Nice, that worked. Thanks!